.github/workflows/ci.yaml

```diff
@@ -13,15 +13,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: ['1.20', '1.21', '1.22']
+        go-version: ['1.21', '1.22']
```

Comment:
you can use 'oldstable', 'stable' if you want, so you never have to update them

https://github.com/marketplace/actions/setup-go-environment#using-stableoldstable-aliases

Reply:
you already used here BTW

https://github.com/mfridman/tparse/pull/119/files#diff-4b122024a3a28ded65da76a2f1bface1f3a27328374438d1298d25585fe0603bR22

Reply:
Updated in https://github.com/mfridman/tparse/pull/119/commits/fb13b85755bc6c499e10c2ca469c3b6b1d78afb4 for consistency.